.env.example

```diff
@@ -1,5 +1,26 @@
 #For more information on .env files, their content and format: https://pypi.org/project/python-dotenv/
 
+######################################
+########### Admin Settings ###########
+######################################
+
+# Enable Admin GUI, available for the first user in LNBITS_ADMIN_USERS if available
+# Warning: Enabling this will make LNbits ignore this configuration file. Your settings will
+# be stored in your database and you will be able to change them only through the Admin UI.
+# Disable this to make LNbits use this config file again.
```

Comment:
Some settings like DB connection, `LNBITS_ADMIN_UI` itself (and others) are always read from the environment variables (`.env` file or actual env vars).

These settings are grouped under the [ReadOnlySettings](https://github.com/lnbits/lnbits/blob/dev/lnbits/settings.py#L401) class.